### PR TITLE
Refactor powerups

### PR DESCRIPTION
**Pull request by Bzygu234**

### CODE REVIEW

**Review by BartoszChadrys — APPROVED** [review]

File: Player.py [hunk 1]
```diff
@@ -174,6 +176,23 @@ def isPlayerNearTheBlock(self, direction: Direction):
         else:
             return False
 
+    def checkForBomb(self, y, x):
+        for bomb in self.bombsHandler.everyPlayersBombs:
+            bombY, bombX = getTileCoordinates(bomb.y, bomb.x)
+            if bombX == x and bombY == y:
+                return True
+        return False
+    def isNextTileABomb(self, y, x, direction: Direction):
```
Comment (olekgolus11) [inline Player.py:185]: and a next line

File: utilities.py [hunk 2]
```diff
@@ -7,7 +8,10 @@
 def getTileCoordinates(y, x):
     return (y + TILE_SIZE // 2) // TILE_SIZE, (x + TILE_SIZE // 2) // TILE_SIZE
 
-
+def generateSeed(n):
+    start = 10 ** (n - 1)
+    end = (10 ** n) - 1
+    return random.randint(start, end)
```
Comment (olekgolus11) [inline utilities.py:14]: 10 ^ 60 is a huge number, isn't it? Do we still need this `MAX_NUMBER_OF_CRATES`?

**Review by olekgolus11 — CHANGES_REQUESTED** [review]

File: utilities.py (hunk 2 above)
Comment (Bzygu234) [reply utilities.py:14]: In fact we don't, already changed that

File: Player.py (hunk 1 above)
Comment (Bzygu234) [reply Player.py:185]: done

File: utilities.py (hunk 2 above)
Comment (olekgolus11) [reply utilities.py:14]: do we even need to pass `n`?

**Review by olekgolus11 — CHANGES_REQUESTED** [review]

File: utilities.py (hunk 2 above)
Comment (Bzygu234) [reply utilities.py]: we don't, already changed that

**Review by olekgolus11 — APPROVED** [review]

**Pull request merged**